Cargo.toml

```diff
@@ -20,12 +20,8 @@ tag-message = "ssh-key-dir v{{version}}"
 
 # See https://github.com/coreos/cargo-vendor-filterer
 [package.metadata.vendor-filter]
-# cargo-vendor-filterer supports wildcards but requires some degree of LLVM
-# support for every matching platform, which isn't necessarily available
-# out of the box.  Use this as a stand-in.  Note that architecture-specific
-# crate dependencies (apart from wasm32) are uncommon in the ecosystem.
-platforms = ["x86_64-unknown-linux-gnu"]
-all-features = true
```

Comment:
While ssh-key-dir doesn't currently have any features, I think it'd be safer to retain this?

Reply:
Good catch thank you!